internal/util/flowgraph/input_node.go

```diff
@@ -73,15 +74,27 @@ func (inNode *InputNode) InStream() msgstream.MsgStream {
 
 // Operate consume a message pack from msgstream and return
 func (inNode *InputNode) Operate(in []Msg) []Msg {
-	msgPack, ok := <-inNode.inStream.Chan()
-	if !ok {
-		log.Warn("MsgStream closed", zap.Any("input node", inNode.Name()))
-		return []Msg{&MsgStreamMsg{
-			isCloseMsg: true,
-		}}
+	var msgPack *msgstream.MsgPack
+	var ok bool
+	select {
```

Comment:
If the `inStream.Chan` has been closed and the `syncChan` received a message, two case will be randomly selected. Maybe the message of the `syncChan` will be ignored.